Packages/Store/Sources/Requests/ContactAddressInfoListRequest.swift

```diff
@@ -0,0 +1,30 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+
+import GRDBQuery
+import GRDB
+import Primitives
+import Combine
+
+public struct ContactAddressInfoListRequest: ValueObservationQueryable {
+    public static var defaultValue: [ContactAddressInfo] { [] }
+    
+    private let chain: Chain
+    
+    public init(chain: Chain) {
+        self.chain = chain
+    }
+    
+    public func fetch(_ db: Database) throws -> [ContactAddressInfo] {
+        do {
+            let entities = try ContactAddressRecord
```

Comment:
```suggestion
            let entities = try ContactAddressRecord
```

just return, no need for `entities`